src/_base/harness/attributes/docker-base.yml

```diff
@@ -37,8 +37,12 @@ attributes:
       services:
         php-base:
           environment:
+            DB_HOST: = '{{ .Values.resourcePrefix }}' ~ @('database.host')
             APP_HOST: = @('pipeline.base.hostname')
             PHP_IDE_CONFIG: = ''
+        nginx:
+          environment:
+            DB_HOST: = '{{ .Values.resourcePrefix }}php-fpm'
```

Comment:
lol, umm, nothing to see here
![](https://media.giphy.com/media/13d2jHlSlxklVe/giphy.gif)